org.filezilla_project.FileZilla.json

```diff
@@ -0,0 +1,132 @@
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "filezilla",
+    "rename-icon": "filezilla",
+    "copy-icon": true,
+    "rename-desktop-file": "filezilla.desktop",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.PowerManagement",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.gnome.SessionManager",
+        "--filesystem=host",
+        "--share=network",
+        "--require-version=0.10.3",
+        "--socket=ssh-auth"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/share/aclocal"
+    ],
+    "modules": [
+        "shared-modules/glu/glu-9.json",
+        {
+            "name": "boost",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://archives.boost.io/release/1.87.0/source/boost_1_87_0.tar.bz2",
+                    "sha256": "af57be25cb4c4f4b413ed692fe378affb4352ea50fbe294a11ef548f4d527d89"
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh",
+                "./b2 install --prefix=/app/ variant=release link=static threading=multi --with-regex -j $FLATPAK_BUILDER_N_JOBS"
+            ],
+            "cleanup": [
+                "/lib/",
+                "/lib/debug",
+                "/include",
+                ""
+            ]
+        },
+        {
+            "name": "wxWidgets",
+            "cleanup": [
+                "/bin",
+                "/share/bakefile"
+            ],
+            "config-opts": [
+                "--with-opengl",
+                "--with-libjpeg",
+                "--with-libtiff",
+                "--with-libpng",
+                "--with-zlib",
+                "--disable-sdltest",
+                "--enable-unicode",
+                "--enable-display",
+                "--enable-propgrid",
+                "--disable-webview",
+                "--disable-webviewwebkit",
+                "--with-expat=builtin",
+                "--with-libiconv=/usr"
+            ],
+            "build-options": {
+                "cxxflags": "-std=c++0x"
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.3/wxWidgets-3.2.3.tar.bz2",
+                    "sha256": "c170ab67c7e167387162276aea84e055ee58424486404bba692c401730d1a67a"
+                }
+            ]
+        },
+        {
+            "name": "libfilezilla",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://deb.debian.org/debian/pool/main/libf/libfilezilla/libfilezilla_0.49.0.orig.tar.xz",
```

Comment:
please use upstream and not a debian source tarball.

Reply:
I tried, but the links the upstream website generates have some kind of key and will stop working after 30 min or such.

Other urls I found, ended up downloading non archive files (likely the website)